### ISSUE TITLE
Podman fails to start container after update to 1.6.0

### ISSUE THREAD

**Issue by petoem** [containers/libpod#4183]
After updating from 1.5.1 to 1.6.0, podman cannot start containers.
Downgrading back to 1.5.1, lets me start containers again.

Any help would be great to get 1.6.0 to work.
```sh
[petoem@notebook:~]$ uname -a                                                                                                                                                                             (10-03 17:29)
Linux notebook 5.3.1-arch1-1-ARCH #1 SMP PREEMPT Sat Sep 21 11:33:49 UTC 2019 x86_64 GNU/Linux

[petoem@notebook:~]$ podman version --log-level debug                                                                                                                                                     (10-03 17:30)
DEBU[0000] using conmon: "/usr/bin/conmon"              
DEBU[0000] Initializing boltdb state at /home/petoem/.local/share/containers/storage/libpod/bolt_state.db 
DEBU[0000] Using graph driver vfs                       
DEBU[0000] Using graph root /home/petoem/.local/share/containers/storage 
DEBU[0000] Using run root /run/user/1000                
DEBU[0000] Using static dir /home/petoem/.local/share/containers/storage/libpod 
DEBU[0000] Using tmp dir /run/user/1000/libpod/tmp      
DEBU[0000] Using volume path /home/petoem/.local/share/containers/storage/volumes 
DEBU[0000] Set libpod namespace to ""                   
DEBU[0000] [graphdriver] trying provided driver "vfs"   
DEBU[0000] Initializing event backend journald          
DEBU[0000] using runtime "/usr/bin/runc"                
WARN[0000] Error initializing configured OCI runtime crun: no valid executable found for OCI runtime crun: invalid argument 
INFO[0000] running as rootless                          
Version:            1.6.0
RemoteAPI Version:  1
Go Version:         go1.13
OS/Arch:            linux/amd64

[petoem@notebook:~]$ podman create --name httpd-test -p 8080:80 --log-level debug docker.io/katacoda/docker-http-server                                                                                   (10-03 17:33)
DEBU[0000] using conmon: "/usr/bin/conmon"              
DEBU[0000] Initializing boltdb state at /home/petoem/.local/share/containers/storage/libpod/bolt_state.db 
DEBU[0000] Using graph driver vfs                       
DEBU[0000] Using graph root /home/petoem/.local/share/containers/storage 
DEBU[0000] Using run root /run/user/1000                
DEBU[0000] Using static dir /home/petoem/.local/share/containers/storage/libpod 
DEBU[0000] Using tmp dir /run/user/1000/libpod/tmp      
DEBU[0000] Using volume path /home/petoem/.local/share/containers/storage/volumes 
DEBU[0000] Set libpod namespace to ""                   
DEBU[0000] [graphdriver] trying provided driver "vfs"   
DEBU[0000] Initializing event backend journald          
DEBU[0000] using runtime "/usr/bin/runc"                
WARN[0000] Error initializing configured OCI runtime crun: no valid executable found for OCI runtime crun: invalid argument 
INFO[0000] running as rootless                          
DEBU[0000] using conmon: "/usr/bin/conmon"              
DEBU[0000] Initializing boltdb state at /home/petoem/.local/share/containers/storage/libpod/bolt_state.db 
DEBU[0000] Using graph driver vfs                       
DEBU[0000] Using graph root /home/petoem/.local/share/containers/storage 
DEBU[0000] Using run root /run/user/1000                
DEBU[0000] Using static dir /home/petoem/.local/share/containers/storage/libpod 
DEBU[0000] Using tmp dir /run/user/1000/libpod/tmp      
DEBU[0000] Using volume path /home/petoem/.local/share/containers/storage/volumes 
DEBU[0000] Set libpod namespace to ""                   
DEBU[0000] Initializing event backend journald          
DEBU[0000] using runtime "/usr/bin/runc"                
WARN[0000] Error initializing configured OCI runtime crun: no valid executable found for OCI runtime crun: invalid argument 
DEBU[0000] parsed reference into "[vfs@/home/petoem/.local/share/containers/storage+/run/user/1000]docker.io/katacoda/docker-http-server:latest" 
DEBU[0000] parsed reference into "[vfs@/home/petoem/.local/share/containers/storage+/run/user/1000]@0d6ee549ae1314ff0c0b8fea18042f3891250ba6b920fc714563be624e3c62a3" 
DEBU[0000] [graphdriver] trying provided driver "vfs"   
DEBU[0000] exporting opaque data as blob "sha256:0d6ee549ae1314ff0c0b8fea18042f3891250ba6b920fc714563be624e3c62a3" 
DEBU[0000] parsed reference into "[vfs@/home/petoem/.local/share/containers/storage+/run/user/1000]@0d6ee549ae1314ff0c0b8fea18042f3891250ba6b920fc714563be624e3c62a3" 
DEBU[0000] exporting opaque data as blob "sha256:0d6ee549ae1314ff0c0b8fea18042f3891250ba6b920fc714563be624e3c62a3" 
DEBU[0000] parsed reference into "[vfs@/home/petoem/.local/share/containers/storage+/run/user/1000]@0d6ee549ae1314ff0c0b8fea18042f3891250ba6b920fc714563be624e3c62a3" 
DEBU[0000] No hostname set; container's hostname will default to runtime default 
DEBU[0000] Using slirp4netns netmode                    
DEBU[0000] setting container name httpd-test            
DEBU[0000] created OCI spec and options for new container 
DEBU[0000] Allocated lock 0 for container 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7 
DEBU[0000] parsed reference into "[vfs@/home/petoem/.local/share/containers/storage+/run/user/1000]@0d6ee549ae1314ff0c0b8fea18042f3891250ba6b920fc714563be624e3c62a3" 
DEBU[0000] exporting opaque data as blob "sha256:0d6ee549ae1314ff0c0b8fea18042f3891250ba6b920fc714563be624e3c62a3" 
DEBU[0000] created container "13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7" 
DEBU[0000] container "13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7" has work directory "/home/petoem/.local/share/containers/storage/vfs-containers/13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7/userdata" 
DEBU[0000] container "13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7" has run directory "/run/user/1000/vfs-containers/13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7/userdata" 
DEBU[0000] New container created "13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7" 
13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7

[petoem@notebook:~]$ podman start --log-level debug httpd-test                                                                                                                                            (10-03 17:33)
DEBU[0000] using conmon: "/usr/bin/conmon"              
DEBU[0000] Initializing boltdb state at /home/petoem/.local/share/containers/storage/libpod/bolt_state.db 
DEBU[0000] Using graph driver vfs                       
DEBU[0000] Using graph root /home/petoem/.local/share/containers/storage 
DEBU[0000] Using run root /run/user/1000                
DEBU[0000] Using static dir /home/petoem/.local/share/containers/storage/libpod 
DEBU[0000] Using tmp dir /run/user/1000/libpod/tmp      
DEBU[0000] Using volume path /home/petoem/.local/share/containers/storage/volumes 
DEBU[0000] Set libpod namespace to ""                   
DEBU[0000] [graphdriver] trying provided driver "vfs"   
DEBU[0000] Initializing event backend journald          
DEBU[0000] using runtime "/usr/bin/runc"                
WARN[0000] Error initializing configured OCI runtime crun: no valid executable found for OCI runtime crun: invalid argument 
INFO[0000] running as rootless                          
DEBU[0000] using conmon: "/usr/bin/conmon"              
DEBU[0000] Initializing boltdb state at /home/petoem/.local/share/containers/storage/libpod/bolt_state.db 
DEBU[0000] Using graph driver vfs                       
DEBU[0000] Using graph root /home/petoem/.local/share/containers/storage 
DEBU[0000] Using run root /run/user/1000                
DEBU[0000] Using static dir /home/petoem/.local/share/containers/storage/libpod 
DEBU[0000] Using tmp dir /run/user/1000/libpod/tmp      
DEBU[0000] Using volume path /home/petoem/.local/share/containers/storage/volumes 
DEBU[0000] Set libpod namespace to ""                   
DEBU[0000] Initializing event backend journald          
DEBU[0000] using runtime "/usr/bin/runc"                
WARN[0000] Error initializing configured OCI runtime crun: no valid executable found for OCI runtime crun: invalid argument 
DEBU[0000] [graphdriver] trying provided driver "vfs"   
WARN[0000] unable to set namespace: "Error switching to ns /proc/2801/task/2808/ns/net: operation not permitted" 
DEBU[0000] Made network namespace at /run/user/1000/netns/cni-c5f1010c-1f97-17ff-1c6e-b19cae0bf8d8 for container 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7 
DEBU[0000] mounted container "13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7" at "/home/petoem/.local/share/containers/storage/vfs/dir/d78fc0744351e964db31ac88e4b636e2b1d3f5c5941148b3e522107b2ff85ea8" 
DEBU[0000] slirp4netns command: /usr/bin/slirp4netns --api-socket /run/user/1000/libpod/tmp/13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7.net --disable-host-loopback --mtu 65520 --enable-sandbox -c -e 3 -r 4 --netns-type=path /run/user/1000/netns/cni-c5f1010c-1f97-17ff-1c6e-b19cae0bf8d8 tap0 
DEBU[0000] Created root filesystem for container 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7 at /home/petoem/.local/share/containers/storage/vfs/dir/d78fc0744351e964db31ac88e4b636e2b1d3f5c5941148b3e522107b2ff85ea8 
INFO[0000] No non-localhost DNS nameservers are left in resolv.conf. Using default external servers: [nameserver 8.8.8.8 nameserver 8.8.4.4] 
INFO[0000] IPv6 enabled; Adding default IPv6 external servers: [nameserver 2001:4860:4860::8888 nameserver 2001:4860:4860::8844] 
DEBU[0000] /etc/system-fips does not exist on host, not mounting FIPS mode secret 
DEBU[0000] Created OCI spec for container 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7 at /home/petoem/.local/share/containers/storage/vfs-containers/13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7/userdata/config.json 
DEBU[0000] /usr/bin/conmon messages will be logged to syslog 
DEBU[0000] running conmon: /usr/bin/conmon               args="[--api-version 1 -c 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7 -u 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7 -r /usr/bin/runc -b /home/petoem/.local/share/containers/storage/vfs-containers/13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7/userdata -p /run/user/1000/vfs-containers/13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7/userdata/pidfile -l k8s-file:/home/petoem/.local/share/containers/storage/vfs-containers/13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7/userdata/ctr.log --exit-dir /run/user/1000/libpod/tmp/exits --socket-dir-path /run/user/1000/libpod/tmp/socket --log-level debug --syslog --conmon-pidfile /run/user/1000/vfs-containers/13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7/userdata/conmon.pid --exit-command /usr/bin/podman --exit-command-arg --root --exit-command-arg /home/petoem/.local/share/containers/storage --exit-command-arg --runroot --exit-command-arg /run/user/1000 --exit-command-arg --log-level --exit-command-arg debug --exit-command-arg --cgroup-manager --exit-command-arg cgroupfs --exit-command-arg --tmpdir --exit-command-arg /run/user/1000/libpod/tmp --exit-command-arg --runtime --exit-command-arg runc --exit-command-arg --storage-driver --exit-command-arg vfs --exit-command-arg --events-backend --exit-command-arg journald --exit-command-arg container --exit-command-arg cleanup --exit-command-arg 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7]"
[conmon:d]: failed to write to /proc/self/oom_score_adj: Permission denied

DEBU[0000] Received: -1                                 
DEBU[0000] Cleaning up container 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7 
DEBU[0000] Tearing down network namespace at /run/user/1000/netns/cni-c5f1010c-1f97-17ff-1c6e-b19cae0bf8d8 for container 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7 
ERRO[0000] unable to cleanup network for container 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7: "error unmounting network namespace for container 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7: failed to unmount NS: at /run/user/1000/netns/cni-c5f1010c-1f97-17ff-1c6e-b19cae0bf8d8: invalid argument" 
DEBU[0000] unmounted container "13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7" 
ERRO[0000] unable to start container "httpd-test": nsenter: failed to setns to /run/user/1000/netns/cni-c5f1010c-1f97-17ff-1c6e-b19cae0bf8d8: Invalid argument
container_linux.go:345: starting container process caused "process_linux.go:303: getting the final child's pid from pipe caused \"EOF\"": OCI runtime error 
[petoem@notebook:~][125]$ 

```

**Comment by TomSweeneyRedHat** [comment]
Can you try `podman system migrate`?  I didn't think it was necessary for this release, but might have missed that proclamation.

**Comment by mheon** [comment]
This looks like a slirp4netns issue that was resolved in 1.6.1

**Comment by petoem** [comment]
@TomSweeneyRedHat I ran `podman system migrate` (output below), it sadly didn't help. 
But I noticed that whenever I reboot, the first command I run with podman panics and the subsequent commands  sometime display an error like this `ERRO[0000] Error refreshing container 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7: error acquiring lock 0 for container 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7: file exists`.

```sh
[petoem@notebook:~]$ podman version --log-level debug                                                                                                                                                     (10-03 19:51)
DEBU[0000] using conmon: "/usr/bin/conmon"              
DEBU[0000] Initializing boltdb state at /home/petoem/.local/share/containers/storage/libpod/bolt_state.db 
DEBU[0000] Using graph driver vfs                       
DEBU[0000] Using graph root /home/petoem/.local/share/containers/storage 
DEBU[0000] Using run root /run/user/1000                
DEBU[0000] Using static dir /home/petoem/.local/share/containers/storage/libpod 
DEBU[0000] Using tmp dir /run/user/1000/libpod/tmp      
DEBU[0000] Using volume path /home/petoem/.local/share/containers/storage/volumes 
DEBU[0000] Set libpod namespace to ""                   
DEBU[0000] Not configuring container store              
DEBU[0000] Initializing event backend journald          
DEBU[0000] using runtime "/usr/bin/runc"                
WARN[0000] Error initializing configured OCI runtime crun: no valid executable found for OCI runtime crun: invalid argument 
DEBU[0000] using conmon: "/usr/bin/conmon"              
DEBU[0000] Initializing boltdb state at /home/petoem/.local/share/containers/storage/libpod/bolt_state.db 
DEBU[0000] Using graph driver vfs                       
DEBU[0000] Using graph root /home/petoem/.local/share/containers/storage 
DEBU[0000] Using run root /run/user/1000                
DEBU[0000] Using static dir /home/petoem/.local/share/containers/storage/libpod 
DEBU[0000] Using tmp dir /run/user/1000/libpod/tmp      
DEBU[0000] Using volume path /home/petoem/.local/share/containers/storage/volumes 
DEBU[0000] Set libpod namespace to ""                   
DEBU[0000] [graphdriver] trying provided driver "vfs"   
DEBU[0000] Initializing event backend journald          
DEBU[0000] using runtime "/usr/bin/runc"                
WARN[0000] Error initializing configured OCI runtime crun: no valid executable found for OCI runtime crun: invalid argument 
DEBU[0000] Initialized SHM lock manager at path /libpod_rootless_lock_1000 
DEBU[0000] Podman detected system restart - performing state refresh 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x564d3f939763]

goroutine 1 [running]:
github.com/containers/libpod/libpod.(*Runtime).getPodNetwork(0xc0002e4000, 0xc0002214c0, 0x40, 0xc000553d30, 0xa, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	libpod/networking_linux.go:31 +0x63
github.com/containers/libpod/libpod.(*Container).refreshCNI(0xc0004958f0, 0x0, 0x0)
	libpod/container_internal_linux.go:1332 +0xe4
github.com/containers/libpod/libpod.(*Container).refresh(0xc0004958f0, 0x564d413b7248, 0x0)
	libpod/container_internal.go:625 +0x3a4
github.com/containers/libpod/libpod.(*Runtime).refresh(0xc0002e4000, 0xc000324800, 0x1f, 0x0, 0x0)
	libpod/runtime.go:1344 +0x17d
github.com/containers/libpod/libpod.makeRuntime(0x564d405462e0, 0xc000046138, 0xc0002e4000, 0x0, 0x0)
	libpod/runtime.go:1225 +0x14d4
github.com/containers/libpod/libpod.newRuntimeFromConfig(0x564d405462e0, 0xc000046138, 0x0, 0x0, 0xc0000100e0, 0x1, 0x1, 0xc0002e4000, 0x0, 0x0)
	libpod/runtime.go:729 +0x708
github.com/containers/libpod/libpod.NewRuntime(...)
	libpod/runtime.go:427
github.com/containers/libpod/cmd/podman/libpodruntime.getRuntime(0x564d405462e0, 0xc000046138, 0xc000529b10, 0x564d01000000, 0xc0002f0000, 0x7f158acf0d98, 0x0)
	cmd/podman/libpodruntime/runtime.go:180 +0xc8d
github.com/containers/libpod/cmd/podman/libpodruntime.GetRuntime(...)
	cmd/podman/libpodruntime/runtime.go:33
main.setupRootless(0x564d4131a460, 0xc000532180, 0x0, 0x2, 0x0, 0x0)
	cmd/podman/main_local.go:165 +0x14b
main.before(0x564d4131a460, 0xc000532180, 0x0, 0x2, 0x1, 0xc00030e888)
	cmd/podman/main.go:124 +0xd2
main.glob..func68(0x564d4131a460, 0xc000532180, 0x0, 0x2, 0x0, 0x0)
	cmd/podman/main.go:78 +0x4b
github.com/spf13/cobra.(*Command).execute(0x564d4131a460, 0xc00003c0e0, 0x2, 0x2, 0x564d4131a460, 0xc00003c0e0)
	vendor/github.com/spf13/cobra/command.go:805 +0x56d
github.com/spf13/cobra.(*Command).ExecuteC(0x564d4131a6e0, 0x6, 0xc000046107, 0x8)
	vendor/github.com/spf13/cobra/command.go:914 +0x2fd
github.com/spf13/cobra.(*Command).Execute(...)
	vendor/github.com/spf13/cobra/command.go:864
main.main()
	cmd/podman/main.go:160 +0xb2

[petoem@notebook:~][2]$ podman ps --all --log-level debug                                                                                                                                                 (10-03 19:51)
DEBU[0000] using conmon: "/usr/bin/conmon"              
DEBU[0000] Initializing boltdb state at /home/petoem/.local/share/containers/storage/libpod/bolt_state.db 
DEBU[0000] Using graph driver vfs                       
DEBU[0000] Using graph root /home/petoem/.local/share/containers/storage 
DEBU[0000] Using run root /run/user/1000                
DEBU[0000] Using static dir /home/petoem/.local/share/containers/storage/libpod 
DEBU[0000] Using tmp dir /run/user/1000/libpod/tmp      
DEBU[0000] Using volume path /home/petoem/.local/share/containers/storage/volumes 
DEBU[0000] Set libpod namespace to ""                   
DEBU[0000] [graphdriver] trying provided driver "vfs"   
DEBU[0000] Initializing event backend journald          
DEBU[0000] using runtime "/usr/bin/runc"                
WARN[0000] Error initializing configured OCI runtime crun: no valid executable found for OCI runtime crun: invalid argument 
DEBU[0000] Podman detected system restart - performing state refresh 
ERRO[0000] Error refreshing container 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7: error acquiring lock 0 for container 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7: file exists 
INFO[0000] running as rootless                          
DEBU[0000] using conmon: "/usr/bin/conmon"              
DEBU[0000] Initializing boltdb state at /home/petoem/.local/share/containers/storage/libpod/bolt_state.db 
DEBU[0000] Using graph driver vfs                       
DEBU[0000] Using graph root /home/petoem/.local/share/containers/storage 
DEBU[0000] Using run root /run/user/1000                
DEBU[0000] Using static dir /home/petoem/.local/share/containers/storage/libpod 
DEBU[0000] Using tmp dir /run/user/1000/libpod/tmp      
DEBU[0000] Using volume path /home/petoem/.local/share/containers/storage/volumes 
DEBU[0000] Set libpod namespace to ""                   
DEBU[0000] No store required. Not opening container store. 
DEBU[0000] Initializing event backend journald          
DEBU[0000] using runtime "/usr/bin/runc"                
WARN[0000] Error initializing configured OCI runtime crun: no valid executable found for OCI runtime crun: invalid argument 
DEBU[0000] Setting maximum workers to 8                 
CONTAINER ID  IMAGE                                         COMMAND  CREATED      STATUS   PORTS                 NAMES
13a1c14093fb  docker.io/katacoda/docker-http-server:latest  /app     2 hours ago  Created  0.0.0.0:8080->80/tcp  httpd-test

[petoem@notebook:~]$ podman system migrate --log-level debug                                                                                                                                              (10-03 19:53)
DEBU[0000] using conmon: "/usr/bin/conmon"              
DEBU[0000] Initializing boltdb state at /home/petoem/.local/share/containers/storage/libpod/bolt_state.db 
DEBU[0000] Using graph driver vfs                       
DEBU[0000] Using graph root /home/petoem/.local/share/containers/storage 
DEBU[0000] Using run root /run/user/1000                
DEBU[0000] Using static dir /home/petoem/.local/share/containers/storage/libpod 
DEBU[0000] Using tmp dir /run/user/1000/libpod/tmp      
DEBU[0000] Using volume path /home/petoem/.local/share/containers/storage/volumes 
DEBU[0000] Set libpod namespace to ""                   
DEBU[0000] Not configuring container store              
DEBU[0000] Initializing event backend journald          
DEBU[0000] using runtime "/usr/bin/runc"                
WARN[0000] Error initializing configured OCI runtime crun: no valid executable found for OCI runtime crun: invalid argument 
INFO[0000] running as rootless                          
DEBU[0000] using conmon: "/usr/bin/conmon"              
DEBU[0000] Initializing boltdb state at /home/petoem/.local/share/containers/storage/libpod/bolt_state.db 
DEBU[0000] Using graph driver vfs                       
DEBU[0000] Using graph root /home/petoem/.local/share/containers/storage 
DEBU[0000] Using run root /run/user/1000                
DEBU[0000] Using static dir /home/petoem/.local/share/containers/storage/libpod 
DEBU[0000] Using tmp dir /run/user/1000/libpod/tmp      
DEBU[0000] Using volume path /home/petoem/.local/share/containers/storage/volumes 
DEBU[0000] Set libpod namespace to ""                   
DEBU[0000] Not configuring container store              
DEBU[0000] Initializing event backend journald          
WARN[0000] Error initializing configured OCI runtime crun: no valid executable found for OCI runtime crun: invalid argument 
DEBU[0000] using runtime "/usr/bin/runc"                
INFO[0000] stopping all containers                      

[petoem@notebook:~]$ podman start --log-level debug httpd-test                                                                                                                                            (10-03 19:54)
DEBU[0000] using conmon: "/usr/bin/conmon"              
DEBU[0000] Initializing boltdb state at /home/petoem/.local/share/containers/storage/libpod/bolt_state.db 
DEBU[0000] Using graph driver vfs                       
DEBU[0000] Using graph root /home/petoem/.local/share/containers/storage 
DEBU[0000] Using run root /run/user/1000                
DEBU[0000] Using static dir /home/petoem/.local/share/containers/storage/libpod 
DEBU[0000] Using tmp dir /run/user/1000/libpod/tmp      
DEBU[0000] Using volume path /home/petoem/.local/share/containers/storage/volumes 
DEBU[0000] Set libpod namespace to ""                   
DEBU[0000] Not configuring container store              
DEBU[0000] Initializing event backend journald          
DEBU[0000] using runtime "/usr/bin/runc"                
WARN[0000] Error initializing configured OCI runtime crun: no valid executable found for OCI runtime crun: invalid argument 
DEBU[0000] using conmon: "/usr/bin/conmon"              
DEBU[0000] Initializing boltdb state at /home/petoem/.local/share/containers/storage/libpod/bolt_state.db 
DEBU[0000] Using graph driver vfs                       
DEBU[0000] Using graph root /home/petoem/.local/share/containers/storage 
DEBU[0000] Using run root /run/user/1000                
DEBU[0000] Using static dir /home/petoem/.local/share/containers/storage/libpod 
DEBU[0000] Using tmp dir /run/user/1000/libpod/tmp      
DEBU[0000] Using volume path /home/petoem/.local/share/containers/storage/volumes 
DEBU[0000] Set libpod namespace to ""                   
DEBU[0000] [graphdriver] trying provided driver "vfs"   
DEBU[0000] Initializing event backend journald          
DEBU[0000] using runtime "/usr/bin/runc"                
WARN[0000] Error initializing configured OCI runtime crun: no valid executable found for OCI runtime crun: invalid argument 
INFO[0000] running as rootless                          
DEBU[0000] using conmon: "/usr/bin/conmon"              
DEBU[0000] Initializing boltdb state at /home/petoem/.local/share/containers/storage/libpod/bolt_state.db 
DEBU[0000] Using graph driver vfs                       
DEBU[0000] Using graph root /home/petoem/.local/share/containers/storage 
DEBU[0000] Using run root /run/user/1000                
DEBU[0000] Using static dir /home/petoem/.local/share/containers/storage/libpod 
DEBU[0000] Using tmp dir /run/user/1000/libpod/tmp      
DEBU[0000] Using volume path /home/petoem/.local/share/containers/storage/volumes 
DEBU[0000] Set libpod namespace to ""                   
DEBU[0000] Initializing event backend journald          
WARN[0000] Error initializing configured OCI runtime crun: no valid executable found for OCI runtime crun: invalid argument 
DEBU[0000] using runtime "/usr/bin/runc"                
WARN[0000] unable to set namespace: "Error switching to ns /proc/1312/task/1321/ns/net: operation not permitted" 
DEBU[0000] Made network namespace at /run/user/1000/netns/cni-e349ad61-5f50-7646-41c8-879a0b55371e for container 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7 
DEBU[0000] [graphdriver] trying provided driver "vfs"   
DEBU[0000] mounted container "13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7" at "/home/petoem/.local/share/containers/storage/vfs/dir/d78fc0744351e964db31ac88e4b636e2b1d3f5c5941148b3e522107b2ff85ea8" 
DEBU[0000] slirp4netns command: /usr/bin/slirp4netns --api-socket /run/user/1000/libpod/tmp/13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7.net --disable-host-loopback --mtu 65520 --enable-sandbox -c -e 3 -r 4 --netns-type=path /run/user/1000/netns/cni-e349ad61-5f50-7646-41c8-879a0b55371e tap0 
DEBU[0000] Created root filesystem for container 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7 at /home/petoem/.local/share/containers/storage/vfs/dir/d78fc0744351e964db31ac88e4b636e2b1d3f5c5941148b3e522107b2ff85ea8 
INFO[0000] No non-localhost DNS nameservers are left in resolv.conf. Using default external servers: [nameserver 8.8.8.8 nameserver 8.8.4.4] 
INFO[0000] IPv6 enabled; Adding default IPv6 external servers: [nameserver 2001:4860:4860::8888 nameserver 2001:4860:4860::8844] 
DEBU[0000] /etc/system-fips does not exist on host, not mounting FIPS mode secret 
DEBU[0000] Created OCI spec for container 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7 at /home/petoem/.local/share/containers/storage/vfs-containers/13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7/userdata/config.json 
DEBU[0000] /usr/bin/conmon messages will be logged to syslog 
DEBU[0000] running conmon: /usr/bin/conmon               args="[--api-version 1 -c 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7 -u 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7 -r /usr/bin/runc -b /home/petoem/.local/share/containers/storage/vfs-containers/13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7/userdata -p /run/user/1000/vfs-containers/13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7/userdata/pidfile -l k8s-file:/home/petoem/.local/share/containers/storage/vfs-containers/13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7/userdata/ctr.log --exit-dir /run/user/1000/libpod/tmp/exits --socket-dir-path /run/user/1000/libpod/tmp/socket --log-level debug --syslog --conmon-pidfile /run/user/1000/vfs-containers/13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7/userdata/conmon.pid --exit-command /usr/bin/podman --exit-command-arg --root --exit-command-arg /home/petoem/.local/share/containers/storage --exit-command-arg --runroot --exit-command-arg /run/user/1000 --exit-command-arg --log-level --exit-command-arg debug --exit-command-arg --cgroup-manager --exit-command-arg cgroupfs --exit-command-arg --tmpdir --exit-command-arg /run/user/1000/libpod/tmp --exit-command-arg --runtime --exit-command-arg runc --exit-command-arg --storage-driver --exit-command-arg vfs --exit-command-arg --events-backend --exit-command-arg journald --exit-command-arg container --exit-command-arg cleanup --exit-command-arg 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7]"
[conmon:d]: failed to write to /proc/self/oom_score_adj: Permission denied

DEBU[0000] Received: -1                                 
DEBU[0000] Cleaning up container 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7 
DEBU[0000] Tearing down network namespace at /run/user/1000/netns/cni-e349ad61-5f50-7646-41c8-879a0b55371e for container 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7 
ERRO[0000] unable to cleanup network for container 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7: "error unmounting network namespace for container 13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7: failed to unmount NS: at /run/user/1000/netns/cni-e349ad61-5f50-7646-41c8-879a0b55371e: invalid argument" 
DEBU[0000] unmounted container "13a1c14093fbe545bd39d7c8de6915c2cbd4303dcaaad4026e0f97c2c7ba5de7" 
ERRO[0000] unable to start container "httpd-test": nsenter: failed to setns to /run/user/1000/netns/cni-e349ad61-5f50-7646-41c8-879a0b55371e: Invalid argument
container_linux.go:345: starting container process caused "process_linux.go:303: getting the final child's pid from pipe caused \"EOF\"": OCI runtime error 
[petoem@notebook:~][125]$                                                                                                                                                                                 (10-03 19:54)

```

**Comment by mheon** [comment]
Definitely looks like issues that were resolved in 1.6.1

**Comment by petoem** [comment]
I have build and installed 1.6.1 and it works now. (all above issues went away) :smile:
